circleci-provision-scripts/android-ndk.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -ex
+
+function install_ndk() {
+    NDK_VERSION=r10d
+    TMP_NDK=/tmp/ndk
+    DIR=android-ndk-$NDK_VERSION
+    FILE=$DIR-linux-x86_64.bin
+
+    mkdir -p $TMP_NDK
+    curl -L -o $TMP_NDK/$FILE https://dl.google.com/android/ndk/$FILE
+
+    pushd $TMP_NDK
+    chmod a+x $FILE
+    ./$FILE
+    mv $DIR /usr/local/android-ndk
+    popd
+
+    echo 'export "ANDROID_NDK=/usr/local/android-ndk"' >> ${CIRCLECI_HOME}/.circlerc
+    rm -rf $TMP_NDK
+}
+
+# I'm not sure how many users actually use fb-adb, so disabling for now.
+#function install_fb_adb() {
```

Comment:
~~ For disabling, rather than uncommenting it out, you can add it as-is, and users can call `circleci-install fb_adb` if they need it.